test/integration/integration_test.go

```diff
@@ -505,6 +506,13 @@ func (i *integrationTest) exerciseIncrementalBuild(tag, imageName string, remove
 	if checkOnBuild {
 		i.fileExists(containerID, "/sti-fake/src/onbuild")
 	}
+
+	if took := time.Since(start); took > docker.DefaultDockerTimeout {
```

Comment:
I wonder whether this approach could/should also be applied to other integration tests - for example (but not limited to) `func (i *integrationTest) exerciseCleanBuild` ?